example/main.zig

```diff
@@ -110,6 +112,22 @@ fn action(button: wio.Button) void {
             wio.messageBox(.warn, "wio", "warning");
             wio.messageBox(.err, "wio", "error");
         },
+        .equals, .kp_plus => {
+            if (last_size) |last| {
+                var new = last;
+                new.width +|= 10;
+                new.height +|= 10;
+                window.setSize(new);
+            }
+        },
+        .minus, .kp_minus => {
+            if (last_size) |last| {
+                var new = last;
+                new.width -|= 10;
+                new.height -|= 10;
+                window.setSize(new);
+            }
+        },
```

Comment:
I would prefer this to be less code, eg. just Ctrl+S => setSize(320x240) is enough to test.